README.md

```diff
@@ -2,24 +2,56 @@
 
 Proposal for syntax to import ES modules with attributes
 
+Proposed by [littledan](https://github.com/littledan) and [xtuc](https://github.com/xtuc).
+
 ## Motivation
 
 https://github.com/w3c/webcomponents/issues/839
 
-### When would the checks happen
+"The author is the only true arbiter of the parse goal of content".
+
+### Semantics
+
+On the web, before any code executes this semantics apply before any code executes.
+
+#### `type`
+
+Check that the mimetype matches what type was specified in the attributes.
```

Comment:
Note, the type in the attributes isn't a mimetype itself, but rather implies a mimetype.